android/resources/all/drawable/splash_background.xml

```diff
@@ -1,4 +1,4 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:drawable="@android:color/black" />
+    <item android:drawable="@android:color/white" />
```

Comment:
FWIW I don't think we really need changes to the splash screens since they're only seen for a moment.